docs/reference/env-variables.rst

```diff
@@ -0,0 +1,204 @@
+.. meta::
+  :description: rocFFT documentation and API reference library
+  :keywords: rocFFT, ROCm, API, documentation, reference, environment variable, environment
+
+********************************************************************
+rocFFT environment variables
+********************************************************************
+
+This section describes the most important rocFFT environment variables,
+which are grouped by functionality.
+
+Logging and debugging
+=====================
+
+The logging and debugging environment variables for rocFFT are collected in the
+following table.
+
+.. list-table::
+    :header-rows: 1
+    :widths: 70,30
+
+    * - **Environment variable**
+      - **Value**
+
+    * - | ``ROCFFT_LAYER``
+        | Controls which logging layers are active using bitmask values.
+      - | ``0``: No logging
+        | ``1``: Trace logging
+        | ``2``: Bench logging
+        | ``4``: Profile logging
+        | ``8``: Plan logging
+        | ``16``: Kernel I/O logging
+        | ``32``: RTC logging
+        | ``64``: Tuning logging
+        | ``128``: Graph logging
+        | Values can be combined with bitwise OR
+
+    * - | ``ROCFFT_LOG_TRACE_PATH``
+        | Specifies file path for trace logging output.
+      - | String path to trace log file
+        | Used when trace logging is enabled via ``ROCFFT_LAYER``
+
+    * - | ``ROCFFT_LOG_BENCH_PATH``
+        | Specifies file path for bench logging output.
+      - | String path to bench log file
+        | Used when bench logging is enabled via ``ROCFFT_LAYER``
+
+    * - | ``ROCFFT_LOG_PROFILE_PATH``
+        | Specifies file path for profile logging output.
+      - | String path to profile log file
+        | Used when profile logging is enabled via ``ROCFFT_LAYER``
+
+    * - | ``ROCFFT_LOG_PLAN_PATH``
+        | Specifies file path for plan logging output.
+      - | String path to plan log file
+        | Used when plan logging is enabled via ``ROCFFT_LAYER``
+
+    * - | ``ROCFFT_LOG_KERNELIO_PATH``
+        | Specifies file path for kernel I/O logging output.
+      - | String path to kernel I/O log file
+        | Used when kernel I/O logging is enabled via ``ROCFFT_LAYER``
+
+    * - | ``ROCFFT_LOG_RTC_PATH``
+        | Specifies file path for runtime compilation logging output.
+      - | String path to RTC log file
+        | Used when RTC logging is enabled via ``ROCFFT_LAYER``
+
+    * - | ``ROCFFT_LOG_TUNING_PATH``
+        | Specifies file path for tuning logging output.
+      - | String path to tuning log file
+        | Used when tuning logging is enabled via ``ROCFFT_LAYER``
+
+    * - | ``ROCFFT_LOG_GRAPH_PATH``
+        | Specifies file path for graph logging output.
+      - | String path to graph log file
+        | Used when graph logging is enabled via ``ROCFFT_LAYER``
```

Comment:
The context here is that the output can be used with graphviz.  The ``128``: Graph logging comment should be combined with ``ROCFFT_LOG_GRAPH_PATH`` and then we should mention graphviz.